src/Native/Unix/System.Security.Cryptography.Native.Apple/pal_trust.c

```diff
@@ -165,57 +164,88 @@ static int32_t EnumerateTrust(SecTrustSettingsDomain domain,
 
 int32_t AppleCryptoNative_StoreEnumerateUserRoot(CFArrayRef* pCertsOut, int32_t* pOSStatusOut)
 {
-    if (pCertsOut != NULL)
-        *pCertsOut = NULL;
+   if (pCertsOut == NULL)
+        return -1;        
```

Comment:
This has a small change in behavior: previously a non-NULL pOSStatusOut would always be initialized to `noErr`, even if `pCertsOut` was NULL, but now it's not.

Reply:
I changed it, but none of the callers ever get to this error case.